docker-compose.madoc.yml

```
@@ -0,0 +1,32 @@
+version: "3"
+services:
+  omeka_ecosystem:
+    image: digirati/madoc-platform:latest
```

Could driven this through environment too possibly:
```suggestion
    image: digirati/madoc-platform:${MADOC_PLATFORM_VERSION:-latest}
```

---

Left as is - the docker-compose file to use is a TF variable so could have different files to control this.